components.yml

```diff
@@ -0,0 +1,36 @@
+sgovServer:
+  name: SGoV Service
+  url: ${URL}/sgov-server
+  meta: {}
+dbServer:
+  name: A graph database
+  url: ${URL}/graphdb/sparql
+  meta: {}
+authServer:
+  name: OIDC Authentication Service
+  url: ${URL}/keycloak/auth/realms/kodi-dev
+  meta: {}
+ontographer:
+  name: ontoGrapher
+  url: ${URL}/modelujeme/v-nástroji/ontographer
+  meta:
+    workspacePath: "?workspace=%WORKSPACE_IRI%"
+termitServer:
+  name: TermIT backend
+  url: ${URL}/termit-backend
+  meta: {}
+termit:
+  name: TermIT
+  url: ${URL}/modelujeme/v-nástroji/termit
+  meta:
+    workspacePath: "#vocabularies?workspace=%WORKSPACE_IRI%"
+missionControl:
+  name: Mission Control
+  url: ${URL}/modelujeme
+  meta: {}
+issueTracker:
+  name: Issue Tracker
+  url: https://github.com/opendata-mvcr/sgov-assembly-line/issues
+  meta:
+    newBug: https://github.com/opendata-mvcr/sgov-assembly-line/issues/new?labels=bug&template=po-adavek-na-opravu.md&title=
```

Comment:
Co mít dev/prod issue templates (budou tam různé verze VL).

Reply:
Záleží, jak moc ty verze budou rozdílné. Já jsem zastánce pushování do produkce ASAP, špíš bych si řekl, že ty issues budou vždy by default na produkci a pokud budeme destovat dev verzi, tak ty testery poprosíme, ať to nějak specifikují. Ale každopádně udělat to můžeme, když už na to teď máme nástoj (= tuhle konfiguraci) :-)

Reply:
souhlas, nekomplikujme si to teď. Víme, že to umíme :-)